app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeActivity.kt

```diff
@@ -206,22 +205,7 @@ class ComposeActivity :
     public override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        val notificationId = intent.getIntExtra(NOTIFICATION_ID_EXTRA, -1)
-        if (notificationId != -1) {
-            // ComposeActivity was opened from a notification, delete the notification
-            val notificationManager = getSystemService(NOTIFICATION_SERVICE) as NotificationManager
-            notificationManager.cancel(notificationId)
-        }
-
-        // If started from an intent then compose as the account ID from the intent.
-        // Otherwise use the active account. If null then the user is not logged in,
-        // and return from the activity.
-        val intentAccountId = intent.getLongExtra(ACCOUNT_ID_EXTRA, -1)
-        activeAccount = if (intentAccountId != -1L) {
-            accountManager.getAccountById(intentAccountId)
-        } else {
-            accountManager.activeAccount
-        } ?: return
+        activeAccount = accountManager.activeAccount ?: return
```

Comment:
+1

I'll do a sweep through the code after this PR has gone in to catch other places where this should be done.